content/influxdb/v2.1/reference/syntax/line-protocol.md

```diff
@@ -26,43 +26,59 @@ It is a text-based format that provides the measurement, tag set, field set, and
 - [Comments](#comments)
 - [Naming restrictions](#naming-restrictions)
 - [Duplicate points](#duplicate-points)
+- [Out of range values](#out-of-range-values)
+- [Parse errors](#parse-errors)
 
-```js
-// Syntax
+#### Syntax
+
+```py
+# Line protocol syntax
 <measurement>[,<tag_key>=<tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=<field_value>] [<timestamp>]
+```
+#### Example
 
-// Example
+```py
+# Line protocol example
 myMeasurement,tag1=value1,tag2=value2 fieldKey="fieldValue" 1556813561098000000
 ```
 
-Lines separated by the newline character `\n` represent a single point
-in InfluxDB. Line protocol is whitespace sensitive.
+## Lines
 
-{{% note %}}
-Line protocol does not support the newline character `\n` in tag or field values.
-{{% /note %}}
+Line protocol consists of zero or more lines, each terminated by the newline character (`\n`).
+A line may represent one of the following:
```

Comment:
Maybe "contain" rather than represent? It feels awkward to say that a blank line represents a blank line.